src/data/icons.json

```diff
@@ -653,7 +657,6 @@
   { "name": "Hospital", "unicode": "E91D" },
   { "name": "Hotel", "unicode": "E824" },
   { "name": "HourGlass", "unicode": "EA03" },
-  { "name": "HumanResources", "unicode": "F240" },
```

Comment:
This will remove the icon from the documentation, right? That's probably for the best, as we don't have a way to indicate deprecated icons in the documentation and would prefer that no one starts using it now.

Reply:
Yeah, removing icons from the JSON will hide them on the site. I think it's the right thing to do in the absence of documenting deprecated icons--though I wonder if we should look into supporting those at some point as well (e.g. with something like `deprecated-icons.json` or something along those lines).